phantazm-neuron-minestom/src/main/java/com/github/phantazmnetwork/neuron/bindings/minestom/entity/NeuralEntity.java

```diff
@@ -54,7 +59,26 @@ public NeuralEntity(@NotNull MinestomDescriptor descriptor, @NotNull UUID uuid,
 
     @Override
     public void preTick(long time) {
-        navigator.tick(time);
+        if (canPathfind()) {
```

Comment:
not necessary to query canPathfind on every tick, it's done in the navigator + null destinations are properly handled now

Reply:
merge stuff, it'll fix itself eventually :tm:
the stuff outside of server and mob that I touched myself are the descriptors iirc